pkg/minishift/oc/oc_test.go

```diff
@@ -144,6 +146,24 @@ func TestFlagExist(t *testing.T) {
 	assert.False(t, flagExist(expectedOptions, "proxy"))
 }
 
+func Test_get_kubeconfig_global_path(t *testing.T) {
+	testdata := []struct {
+		KubeConfig string
+		Expected   string
+	}{
+		{KubeConfig: "/tmp/john/config", Expected: "/tmp/john/config"},
```

Comment:
what's about adding some more boundary conditions? What does things like ":/home/john" do or  ";;", etc. The tests are only covering the happy paths.

On another note, maybe the split character should be determined at runtime based on the OS, aka ":" on darwin/linux and ';' on windows. That would be more accurate imo.